.idea/gradle.xml

```diff
@@ -8,7 +8,15 @@
         </compositeConfiguration>
         <option name="distributionType" value="DEFAULT_WRAPPED" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
+        <option name="modules">
```

Comment:
I'm assuming this was an accident - why is this file not caught by the .gitignore?